docs/codeql/ql-language-reference/ql-language-specification.rst

```diff
@@ -71,23 +71,14 @@ language="java"/>``).
 A ``qlpack.yml`` file defines a :ref:`QL pack <about-ql-packs>`.
 The content of a ``qlpack.yml`` file is described in the CodeQL CLI documentation. This file
 will not be recognized when using legacy tools that are not based
-on the CodeQL CLI (that is, LGTM.com, LGTM Enterprise, ODASA, CodeQL for
-Eclipse, and CodeQL for Visual Studio).
+on the CodeQL CLI (that is, LGTM.com and LGTM Enterprise).
```

Comment:
LGTM (both forms) now uses CodeQL.
```suggestion
on the CodeQL CLI.
```

Reply:
Perhaps we should just get rid of the entire sentences -- I don't think we have any active products where the QL engine is _not_ CLI based these days. (Except for tail-end support of old LGTM-e releases, but the language specification doesn't completely apply to them anyway).

Reply:
As suggested, I've removed this sentence.